torch/testing/_internal/common_modules.py

```diff
@@ -1641,11 +1644,12 @@ def module_inputs_torch_nn_TransformerEncoder(module_info, device, dtype, requir
         if 'src_mask' in forward_input.kwargs:
             forward_input.kwargs['mask'] = forward_input.kwargs['src_mask']
             del forward_input.kwargs['src_mask']
-        yield ModuleInput(
```

Comment:
The yield pattern here is relatively common for OpInfo. You might need a list() if you need to traverse the result multiple times.